apps/juicebox_stream/lib/juicebox_stream/stream/server.ex

```diff
@@ -138,8 +145,16 @@ defmodule JuiceboxStream.Stream.Server do
     {:reply, {:ok, id}, state}
   end
 
-  def handle_cast({:vote, track_id}, state) do
-    {:noreply, Control.vote(state, track_id)}
+  def handle_call({:vote_up, track_id}, _from, state) do
+    new_state = Control.vote_up(state, track_id)
+    broadcast(state.id, @actions[:QUEUE_UPDATED], %{ videos: new_state.queue })
```

Comment:
There is no whitespace around parentheses/brackets most of the time, but here there is.